contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/MarkdownEditor/MarkdownEditor.vue

```diff
@@ -365,7 +406,7 @@
         // select all (ctrl+a), copy (ctrl+c),
         // cut (ctrl+x), paste (ctrl+v)
         // Disable all remaining default keyboard shortcuts.
-        if (event.ctrlKey === true && ['b', 'i', 'a', 'c', 'x', 'v'].includes(event.key)) {
+        if (event.ctrlKey === true && ['b', 'i', 'a', 'c', 'x', 'v', 'z'].includes(event.key)) {
```

Comment:
I added `ctrl-z`.  Any reason why we weren't allowing that?  It seems pretty useful if you ask me.

Reply:
Yes, we've decided to not support undo/redo actions because there were plans to add history feature for all updates of an assessment item. I am not sure if it still makes sense though, might be good to check it with @rtibbles and @jayoshih.